OpenRA-OpenAL-CS.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>OpenRA-OpenAL-CS</id>
+    <version>0.0.1</version> <!-- Version is set during packaging. -->
+    <authors>OpenRA</authors>
+    <owners>OpenRA</owners>
+    <projectUrl>https://github.com/OpenRA/OpenAL-CS</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>OpenAL-CS modified for OpenRA</description>
+    <copyright>
+    - Copyright (c) 2014-2015 Ethan Lee
+    - Copyright (c) Creative Labs
+    </copyright>
+    <license type="expression">LGPL-2.0-only</license>
```

Comment:
This is again wrong as the wrapper is zlib. I can give you access to my old https://www.nuget.org/packages/OpenAL-Soft so you can separate it properly.